### PR TITLE
OPSEXP-2486 Rewrite ingress, dns and https docs for eks deployment

### PR DESCRIPTION
**Pull request by gionn**
* install ingress-nginx in the quickstart way
* drop external dns in favour of a simple dns creation step
* add cert-manager to generate letsencrypt certs

Ref: OPSEXP-2486

### CODE REVIEW

File: docs/helm/eks-deployment.md [hunk 1]
```diff
@@ -54,138 +54,27 @@ Set the cluster name in an environment variable that can be reused later:
...
 ```
 
-Create the cluster using a supported version (we are currently testing against
-1.24). Most common choices for instance types are `m5.xlarge` and `t3.xlarge`:
+Create the cluster using the latest supported version, check the main [README](../../README.md).
+Most common choices for instance types are `m5.xlarge` and `t3.xlarge`:
 
 ```sh
-eksctl create cluster --name $EKS_CLUSTER_NAME --version 1.24 --instance-types t3.xlarge
+eksctl create cluster --name $EKS_CLUSTER_NAME --version 1.29 --instance-types t3.xlarge --nodes 3
```
Comment (alxgomz) [inline docs/helm/eks-deployment.md:61]: Shouldn't it be 1.28 as we state we test only with EKS 1.28?

File: docs/helm/eks-deployment.md [hunk 2]
```diff
@@ -197,10 +86,11 @@ you can alternatively:
 
 * Use an Elastic File System, installing the ([EFS CSI driver](#efs-csi-driver))
   (the default we suggest, see `storageClass="nfs-client"` values in [helm
-  install section](#latest-enterprise-version))
-* Use an EBS block-storage, enabling [EBS CSI driver](#ebs-csi-driver) (not
-  possible with Enterprise in clustered mode)
-* Use a bucket on [S3](examples/with-aws-services.md#s3)
+  install section](#latest-enterprise-version) and required when repository
+  replicas > 1)
```
Comment (alxgomz) [inline docs/helm/eks-deployment.md:90]:
```suggestion
* Use an Elastic File System, installing the [EFS CSI driver](#efs-csi-driver) (this is what
  is documented below)
```

File: docs/helm/eks-deployment.md [hunk 3]
```diff
@@ -197,10 +86,11 @@ you can alternatively:
...
+* Use an EBS block-storage, enabling [EBS CSI driver](#ebs-csi-driver) (when
+  repository replicas = 1)
```
Comment (alxgomz) [inline docs/helm/eks-deployment.md:92]:
```suggestion
* Use an EBS block-storage, enabling [EBS CSI driver](#ebs-csi-driver) (only
  valid if there's a single replica and if pods cannot be scheduled in different
  availability zones. Avoid in production)
```

File: docs/helm/eks-deployment.md [hunk 4]
```diff
@@ -307,150 +197,182 @@ Create the IAM Service Account with access to EBS that will be used by the drive
...
+Set an environment variable with the hostname which will be useful later.
+
+```sh
+export ACS_HOSTNAME=acs.YOUR-DOMAIN-NAME
+```
+
+### HTTPS
 
-    > NOTE: The command will wait until the deployment is ready so please be patient.
+The most simple way to access applications running on Kubernetes behind HTTPS is
```
Comment (alxgomz) [inline docs/helm/eks-deployment.md:274]:
```suggestion
The simplest way to access applications running on Kubernetes behind HTTPS is
```

File: docs/helm/eks-deployment.md [hunk 5]
```diff
@@ -460,66 +382,85 @@ credentials to access the Alfresco Enterprise registry.
...
+
+If it doesn't happen, you can first describe the pod not in Running state yet
+and look for the events section at the end:
+
+```sh
+kubectl describe pod acs-alfresco-repository-???-??? -n alfresco
+```
+
+If the pod is in running state but can't achieve the `1/1` Ready before the
+liveness timeout kicks in, you should take a look at the logs with:
```
Comment (alxgomz) [inline docs/helm/eks-deployment.md:459]:
```suggestion
readiness probe fails as many times the readiness threshold allows, you should
take a look at the logs with:
```

**Review by alxgomz — CHANGES_REQUESTED** [review]
Good job!

File: docs/helm/eks-deployment.md (hunk 1 above)
Comment (gionn) [reply docs/helm/eks-deployment.md:61]: strictly speaking yes, but given that I tested it manually during this PR and we are testing 1.29 on KinD... we could stay with that

**Review by alxgomz — APPROVED** [review]

File: docs/helm/eks-deployment.md [hunk 6]
```diff
@@ -307,150 +197,182 @@ Create the IAM Service Account with access to EBS that will be used by the drive
...
+See [ingress-nginx](ingress-nginx.md) section.
 
-```bash
-kubectl create namespace alfresco
+### DNS
+
+In order to access Alfresco once installed, you need to setup a DNS record that
+resolve to the ELB hostname that has been provisioned by `ingress-nginx`.
+
+This is a typical architecture for which you can learn more at
```
Comment (pmacius) [inline docs/helm/eks-deployment.md:239]:
```suggestion
This is typical architecture, which you can learn more about at
```

**Review by pmacius — APPROVED** [review]
Im not sure only about this sentence. You can pick which one sounds better 😄

**Review by anxumalo — COMMENTED** [review]
LGTM 👍🏼

**Review by anxumalo — APPROVED** [review]

**Pull request merged**